tests/integ/test_udf.py

```diff
@@ -673,8 +674,8 @@ def plus4_then_mod5_with_import(x):
     session.add_import(stage_file)
     plus4_then_mod5_udf = udf(
         plus4_then_mod5_with_import,
-        return_type=IntegerType(),
-        input_types=[IntegerType()],
+        return_type=LongType(),
+        input_types=[LongType()],
```

Comment:
Is this expected? would this be considered bcr?

Reply:
+1, I think we should avoid changing existing live test code.
if needed I think we can use the local testing mode fixture.

Reply:
The df that is generated below this definition will have an inferred type of LongType. These tests have been relying on the server side coercing the type to IntegerType. This breaks in local testing due to the bug I've described in SNOW-1625599. I could try to fix that bug before merging this change.